pyproject.toml

```diff
@@ -123,6 +123,7 @@ cohere = { version = "^4.56", optional = true }
 pytest = { version = "^7", optional = true }
 pytest-asyncio = { version = "^0.23.0", optional = true }
 mock = { version = "^5", optional = true }
+notion-client = "^2.2.1"
```

Comment:
it shouldn't be put under test part, also need to be set in other group

Reply:
It is under tools and all